### PR TITLE
Fix respec issues

### PR DESCRIPTION
**Pull request by siusin**
- fix ReSpec errors and warnings;
- add links to GitHub issues;

***
<a href="https://pr-preview.s3.amazonaws.com/siusin/resource-timing/pull/135.html" title="Last updated on Jan 23, 2018, 2:13 PM GMT (4dc573b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/resource-timing/135/3c7527b...siusin:4dc573b.html" title="Last updated on Jan 23, 2018, 2:13 PM GMT (4dc573b)">Diff</a>

### CODE REVIEW

**Comment by siusin** [comment]
Done.

Thanks for the review, @marcoscaceres .

**Comment by marcoscaceres** [comment]
Thanks for doing all the hard work @siusin!

File: tidyconfig.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+char-encoding: utf8
+indent: yes
```
Comment (igrigorik) [inline tidyconfig.txt]: FWIW, we use auto in other specs. E.g. https://github.com/w3c/preload/blob/gh-pages/tidyconfig.txt

**Comment by igrigorik** [comment]
@siusin thanks for all the great work here! Apologies, looks like we landed some updates and now have a conflict. Any chance you can sync this PR? Otherwise, just one minor nit on the tidyconfig.

File: index.html [hunk 2]
```diff
@@ -1126,14 +1100,14 @@ <h3>Processing Model</h3>
...
 to ensure timing attributes are not skewed by adjustments to the
-system clock while <a data-cite="!FETCH/#concept-fetch"
-title='fetch'>fetching</a> the resource. The difference between any
-two chronologically recorded timing attributes MUST never be
-negative. For all resources, including subdocument resources, the
-user agent MUST record the system clock at the beginning of the
-root document navigation and define subsequent timing attributes in
-terms of a monotonic clock measuring time elapsed from the
-beginning of the navigation.</p>
+system clock while <a data-cite="!FETCH/#concept-fetch" title=
```
Comment (marcoscaceres) [inline index.html]: Nit: replace `title=` with `data-lt=`.

File: index.html [hunk 3]
```diff
@@ -866,18 +842,18 @@ <h3>Extensions to the <code>Performance</code> Interface</h3>
 <li>Add <i>new entry</i> to the <a data-cite=
 '!PERFORMANCE-TIMELINE-2/#dfn-performance-entry-buffer'>performance
 entry buffer</a>.</li>
-<li>increase <a>resource timing buffer current size</a> by 1.</li>
+<li>ncrease <a>resource timing buffer current size</a> by 1.</li>
```
Comment (marcoscaceres) [inline index.html]: nit: typo

File: index.html [hunk 4]
```diff
@@ -207,15 +207,15 @@ <h2>Terminology</h2>
 <p>The term <dfn>resource</dfn> is used to refer to elements and
 any other user-initiated fetches throughout this specification. For
 example, a resource could originate from XMLHttpRequest objects
-[[XHR]], HTML elements [[HTML5]] such as iframe, img,
-script, object, embed, and link with the link type of stylesheet,
-and SVG elements [[SVG11]] such as svg.</p>
+[[XHR]], HTML elements [[HTML5]] such as iframe, img, script,
```
Comment (marcoscaceres) [inline index.html]: nit: s/HTML/HTML5

File: index.html (hunk 2 above)
Comment (marcoscaceres) [reply index.html]: Actually, just remove title entirely.

File: index.html (hunk 3 above)
Comment (siusin) [reply index.html]: Oops, done :)

File: index.html (hunk 4 above)
Comment (siusin) [reply index.html]: You mean s/[[HTML5]]/[[HTML]]? I've made this change in the latest commit. Thanks!

Comment (marcoscaceres) [reply index.html]: Awesome, thanks!

**Comment by marcoscaceres** [comment]
Once the merge conflict is resolved, would be good to go through the spec and remove all the unnecessary boilerplate (e.g., the conformance section talks about IDL fragment conformance, which is unnecessary... there is a ton of stuff like that... the examples are also a bit "c++"... could make them idiomatic JS). 

We can also fix a bunch of markup by using `<dfn data-cite=>` once and then just `<a>` to those defined things.

**Comment by siusin** [comment]
Ok @igrigorik . I'd prefer to merge this PR before improving the examples and the conformance section.

Thanks again for the good suggestions @marcoscaceres .

**Pull request merged**